src/components/atoms/Spacer/Spacer.tsx

```diff
@@ -0,0 +1,9 @@
+interface SpacerProps {
+  space: number;
+}
+
+const Spacer: React.FC<SpacerProps> = ({ space }) => {
+  return <div style={{ width: '1px', marginBottom: `${space}px` }} />;
+};
```

Comment:
이런 공용 컴포넌트를 만든다면 당장 동적으로 필요한 값이 `marginBottom` 뿐이라 해도, 각 CSS 속성에 매칭되는 명확한 prop명을 사용해주는 것이 좋습니다.

`<Spacer>`를 사용하는 곳에서 `space`라는 prop을 넘겨줄 때, 이 `space`가 `marginBottom`에 사용되리라고는 생각하기 어려울 테니까요. 

수정하신다면 `width`값도 prop로 받되, optional로 설정한 후 default value를 1px로 주는 것이 확장성 측면에서 더 좋습니다

Reply:
지그의 피드백에 동의합니다! `<Spacer>`는 명칭상 가로, 세로 공간을 확보하는지 알 수 없습니다.

다음과 같이 사용 방법이 명확하고, 범용성 있는 컴포넌트로 리팩토링 하였습니다!

```suggestion
const Spacer: React.FC<SpacerProps> = ({ height = 1, width = 1 }) => {
  return <div style={{ width: `${width}px`, marginBottom: `${height}px` }} />;
};
```